addons/mail/static/src/core/common/message_model.js

```diff
@@ -80,10 +80,12 @@ export class Message extends Record {
     incoming_email_cc;
     /** @type {Array[Array[string]]} */
     incoming_email_to;
-    /** @type {boolean} */
-    is_discussion;
-    /** @type {boolean} */
-    is_note;
+    get isDiscussion() {
```

Comment:
I wonder if we shouldn't keep snake case to reduce the diff.
But it's ok too like this.